scripts/build-docker-images

```diff
@@ -41,7 +41,6 @@ done
 cp -r "${CONTRACTS_DIR}" ${WORKDIR}
```

Comment:
Do we still need this?

Reply:
Removed in [6d5f487](https://github.com/EspressoSystems/espresso-sequencer/pull/1262/commits/6d5f48768aa02fbfa2523e3c8c81719dc5549961)